src/api.js

```diff
@@ -4,10 +4,17 @@ const redcarNewsApi = axios.create({
   baseURL: "https://redcar-news.herokuapp.com/api",
 });
 
-export const getArticles = (topic) => {
-  return redcarNewsApi
-    .get("/articles", { params: { topic } })
-    .then(({ data }) => data);
+export const getArticles = (topic, order, sortBy) => {
+  if (sortBy) {
+    const sort_by = sortBy;
+    return redcarNewsApi
+      .get("/articles", { params: { topic, order, sort_by } })
+      .then(({ data }) => data);
+  } else {
+    return redcarNewsApi
+      .get("/articles", { params: { topic, order } })
+      .then(({ data }) => data);
+  }
 };
```

Comment:
Nice use of params here to make the query Abdul. You should be able to pass through all of your params and allow them to be handled by the back end if they exist, meaning a single query.